code/frameworks/angular/src/server/angular-cli-webpack-13.x.x.ts

```diff
@@ -1,31 +1,26 @@
-// Private angular devkit stuff
-const {
-  generateI18nBrowserWebpackConfigFromContext,
-} = require('@angular-devkit/build-angular/src/utils/webpack-browser-config');
-const {
+import { generateI18nBrowserWebpackConfigFromContext } from '@angular-devkit/build-angular/src/utils/webpack-browser-config';
+import {
   getCommonConfig,
   getStylesConfig,
   getDevServerConfig,
-  getTypeScriptConfig,
-} = require('@angular-devkit/build-angular/src/webpack/configs');
-const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+} from '@angular-devkit/build-angular/src/webpack/configs';
+import { TsconfigPathsPlugin } from 'tsconfig-paths-webpack-plugin';
 
-const { filterOutStylingRules } = require('./utils/filter-out-styling-rules');
+import { JsonObject } from '@angular-devkit/core';
+import { BuilderContext } from '@angular-devkit/architect';
+import { filterOutStylingRules } from './utils/filter-out-styling-rules';
 
-/**
- * Extract webpack config from angular-cli 13.x.x
- * ⚠️ This file is in JavaScript to not use TypeScript. Because current storybook TypeScript version is not compatible with Angular CLI.
- * FIXME: Try another way with TypeScript on future storybook version (7 maybe 🤞)
- *
- * @param {*} baseConfig Previous webpack config from storybook
- * @param {*} options { builderOptions, builderContext }
- */
-exports.getWebpackConfig = async (baseConfig, { builderOptions, builderContext }) => {
+export const getWebpackConfig = async (
+  baseConfig: any,
+  { builderOptions, builderContext }: { builderOptions: JsonObject; builderContext: BuilderContext }
+) => {
   /**
    * Get angular-cli Webpack config
    */
   const { config: cliConfig } = await generateI18nBrowserWebpackConfigFromContext(
     {
+      // todo this field was missing according to its interface -> figure out what to do
+      tsConfig: '',
```

Comment:
If i'm not wrong : 

This is used by generateWebpackConfig to get tsConfig options

https://github.com/angular/angular-cli/blob/ea923d0b7bb34826efffc00b21ee0e220000d3b3/packages/angular_devkit/build_angular/src/utils/webpack-browser-config.ts#L43

witch interface have you looked ? 

normally this is the interface contain this field :
https://github.com/angular/angular-cli/blob/13.0.x/packages/angular_devkit/build_angular/src/builders/browser/schema.json#L23